game/src/test/scala/io/aigar/game/CellSpec.scala

```diff
@@ -58,6 +58,16 @@ class CellSpec extends FlatSpec with Matchers {
     small.acceleration.magnitude should be > big.acceleration.magnitude
   }
 
+  it should "not move slower than the minimal speed no matter its mass" in {
```

Comment:
Rewording suggestion:
`not move slower than the minimal speed limit, no matter what its mass is`